src/index.html

```diff
@@ -1,13 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div data-qa="card" class="product-card">
+      <img
+        src="./images/imac.jpeg"
+        alt="image of monoblock apple"
+        class="product-card__image"
+      >
+      <h2 class="product-card__name">
+        Моноблок APPLE A1419 iMac 27'' Retina 5K (MNED2UA/A)
+      </h2>
+      <p class="product-card__product-code">
+        Код товара: 195434
+      </p>
+      <div class="product-card__rating">
+        <div class="stars">
```

Comment:
```suggestion
        <div class="stars stars--4">
```

Let's create `stars--4` modifier that will describe how many active stars should be displayed (use `:nth-child(-n+4)`